### ISSUE TITLE
ListView constructor's addSemanticIndexes argument is undocumented

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#23342]

**Comment by Hixie** [comment]
Looks like it should have been added to the last paragraph of the ListView constructor, like the others.
Also, the documentation for SliverChildListDelegate doesn't say it can't be null.
Also, the SliverChildListDelegate.addSemanticIndexes documentation refers to "wrapping the correct child widgets in an indexed child semantics widget" but doesn't explain what that means.
